.github/workflows/ci.yaml

```diff
@@ -81,19 +80,15 @@ jobs:
           body-template: ${{  steps.create-unchecked-pr-body.outputs.body }}
           body-update-action: replace
 
-  build:
-    name: Build website
+  test:
+    name: Run unit tests
     runs-on: ubuntu-20.04
-    needs: is-safe-to-run
     env:
       CI: true
     steps:
       - id: checkout
         name: Check out source code
         uses: actions/checkout@v3
-        with:
-          ref: ${{github.head_ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
If you remove this, it won't checkut the latest code due to the difference in how `pull_request_target` and `pull_request` event triggers work. Perhaps we could have a brief call on how this works and why we need it.

Reply:
Also importnat for you to keep in mind, with `pull_request_target`, changes to `ci.yaml` will only apply after merging into `master`, so there is no way to actually test this before merging as far as I know